ext/drcovlib/modules.c

```diff
@@ -467,8 +467,9 @@ module_read_entry_print(module_read_entry_t *entry, uint idx, char *buf, size_t
     len = dr_snprintf(buf, size,
                       "%3u, %3u, " PFX ", " PFX ", " PFX ", " ZHEX64_FORMAT_STRING
                       ", " PFX ", ",
-                      idx, entry->containing_id, entry->base, entry->base + entry->size,
-                      entry->entry, entry->offset, entry->preferred_base);
+                      idx, entry->containing_id, entry->base,
+                      (app_pc)((ptr_uint_t)entry->base + entry->size), entry->entry,
```

Comment:
s/`entry->size`/`(ptr_uint_t)entry->size` to avoid:
```
error: cast to pointer from integer of different size [-Werror=int-to-pointer-cast]
```
This is the reason some Github CI are failing.

Reply:
Thanks!